tests/unit/drivers/vector/test_pgvector_vector_store_driver.py

```diff
@@ -33,7 +33,7 @@ def test_initialize_requires_engine_or_connection_string(self, embedding_driver)
 
     def test_initialize_accepts_engine(self, embedding_driver):
         engine = create_engine(self.connection_string)
-        driver = PgVectorVectorStoreDriver(embedding_driver=embedding_driver, engine=engine, table_name=self.table_name)
+        driver = PgVectorVectorStoreDriver(embedding_driver=embedding_driver, engine=engine, table_name=self.table_name)  # type: ignore
```

Comment:
Reason for the type ignore?

Reply:
It seems pyright doesn't like the mock engine. Is there another way to satisfy pyright?

```
/Users/zach/workplace/Griptape/griptape/tests/unit/drivers/vector/test_pgvector_vector_store_driver.py:36:86 - error: Argument of type "MockConnection | Unknown" cannot be assigned to parameter "engine" of type "Engine | None" in function "__init__" (reportGeneralTypeIssues)
```

Reply:
Actually it looks like typing the mock engine as `Any` does the trick:

```
engine: Any = create_engine(self.connection_string)
```

Updated